src/tests/performVerifySorting.spec.js

```diff
@@ -0,0 +1,41 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { login, password } = require('../auth/user.json');
+
+test.describe('Inventory Sorting', () => {
+    const sortParamsName = ['az', 'za'];
+    const sortParamsPrise = ['lohi', 'hilo'];
+    
+    test.beforeEach(async ({ inventoryPage, loginPage }) => {
+        await inventoryPage.navigate();
+        await loginPage.performLogin(login, password);
+    });
+
+    sortParamsName.forEach(sortParam => {
```

Comment:
fantastic!
now let's combine those 2 `test` into single `test`.

You can create expected results in beforeEach functions and perform verification by comparing testData (object that contains sortParam and expected result)